about.markdown

```diff
@@ -13,9 +14,13 @@ La Madera area just west of SPCE on Stagecoach Rd.  This registered 501c3
 non-profit rescues and re-homes a significant number of Siberian Huskies,
 and other related canines in addition to saving horses that are in jeopardy
 of abandonment or euthanasia.  They annually place approximately 10 to 15
-huskies annually to new “forever homes” in the East Mountain area.
+Huskies to new “forever homes” in the East Mountain area.
 
-Cheryl and Terri host a husky pack that varies from about 30 to 40 huskies.
+Cheryl and Terri host a Husky pack that varies from about 30 to 40 Huskies.
```

Comment:
Not sure you need to capitalize the word husky.

Reply:
I hope it doesn't hurt your feelings to leave it capitalized